delta/app/src/main/resources/database/cassadra.ddl

```diff
@@ -0,0 +1,129 @@
+-- CREATE KEYSPACE IF NOT EXISTS delta WITH replication = {'class': 'NetworkTopologyStrategy', 'datacenter1' : 3 };
+-- CREATE KEYSPACE IF NOT EXISTS delta_snapshot WITH replication = {'class': 'NetworkTopologyStrategy', 'datacenter1' : 3 };
+
+CREATE KEYSPACE IF NOT EXISTS delta
```

Comment:
In sourcing, there is already some cassandra statements, we should try to group them no ?